Student/StudentUnitTests/Submissions/SubmissionDetails/SubmissionDetailsPresenterTests.swift

```diff
@@ -20,29 +20,34 @@ import Core
 import TestsFoundation
 
 class SubmissionDetailsView: SubmissionDetailsViewProtocol {
-    var didReload = false
-    var reloaded = XCTestExpectation(description: "Reloaded")
+    var color: UIColor?
+    var navigationController: UINavigationController?
+    let navigationItem = UINavigationItem(title: "Test")
+    var titleSubtitleView = TitleSubtitleView()
+
     var didEmbed = false
-    var embedded = XCTestExpectation(description: "Embedded")
-    var didReloadNavbar = false
-    var reloadedNavbar = XCTestExpectation(description: "Reloaded Navbar")
+    var embedded: UIViewController?
+    func embed(_ controller: UIViewController?) {
+        didEmbed = true
+        embedded = controller
+    }
 
+    var didEmbedInDrawer = false
+    var embeddedInDrawer: UIViewController?
+    func embedInDrawer(_ controller: UIViewController?) {
+        didEmbedInDrawer = true
+        embeddedInDrawer = controller
+    }
+
+    var didReload = false
     func reload() {
         didReload = true
-        reloaded.fulfill()
     }
 
+    var didReloadNavbar = false
```

Comment:
I don't know why but I, too, like the properties right next to the functions like this.